cmake/Modules/legion_helpers.cmake

```diff
@@ -0,0 +1,52 @@
+#=============================================================================
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(get_legion_and_realm_includes _out_var)
```

Comment:
Could we call this 'get_legion_and_realm_file_includes' to make it a little bit clearer that this is adding '-include <file>' and not '-I <directory>'?